managed/services/backup/pitr_storage_service.go

```diff
@@ -0,0 +1,387 @@
+// Copyright (C) 2022 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+package backup
+
+import (
+	"context"
+	"path"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+
+	"github.com/percona/pmm/managed/models"
+	"github.com/percona/pmm/managed/services/minio"
+)
+
+const (
+	// PITRfsPrefix is a prefix (folder) for PITR chunks on the storage
+	PITRfsPrefix = "pbmPitr"
+)
+
+var errUnsupportedLocation = errors.New("unsupported location config")
+
+// PITRStorageService helps perform file lookups in a backup storage location
+type PITRStorageService struct {
```

Comment:
I propose to rename it just to `PIRTService`, because it doesn't store anything here.

Reply:
Maybe something more obvious, e.g. `PIRTTimerangeService` or `PIRTParseService`?